### PR TITLE
Add DeleteSession Command 

### PR DESCRIPTION
**Pull request by chuayijing**

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -157,6 +157,7 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredModuleList(Predicate<Module> predicate);
+
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/model/Model.java:160]: Probably shouldn't add blank lines in other parts of the code else you might create merge conflicts 😳!

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -132,9 +132,11 @@ public void addModule(Module module) {
         addressBook.addModule(module);
     }
 
+
     @Override
     public void setModule(Module target, Module editedModule) {
         requireAllNonNull(target, editedModule);
+
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/model/ModelManager.java:139]: Same thing about the blank lines.

File: src/main/java/seedu/address/model/module/UniqueModuleList.java [hunk 3]
```diff
@@ -83,9 +83,9 @@ public Module getModule(Module module) {
     }
 
     /**
-    * Gets module with given module code.
-    * Returns null if no such module exists.
-    */
+     * Gets module with given module code.
+     * Returns null if no such module exists.
+     */
```
Comment (PotatoCombat) [inline src/main/java/seedu/address/model/module/UniqueModuleList.java:88]: Ehh hope this works but @aakanksha-rai just check that IntelliJ is indenting your comments correctly?

File: src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java [hunk 4]
```diff
@@ -258,16 +258,6 @@ public boolean hasModule(Module module) {
...
-
-        @Override
-        public void deleteModule(Module module) {
-            throw new AssertionError("This method should not be called.");
-        }
-
-        @Override
-        public Module getModule(Module module) {
-            throw new AssertionError("This method should not be called.");
-        }
```
Comment (PotatoCombat) [inline src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java:270]: Err did you have to delete this?

**Review by PotatoCombat — DISMISSED** [review]

**Comment by PotatoCombat** [comment]
It would be good if you could also split your code into smaller commits? Make sure that each commit passes check style and test case too!

File: src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java (hunk 4 above)
Comment (chuayijing) [reply src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java:270]: eh during conflict resolve, there was duplicate, hence i just removed the duplicate

Comment (PotatoCombat) [reply src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java:270]: Hmmm just took a look this and realised its another issue. Sorry about this.

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (PotatoCombat) [reply src/main/java/seedu/address/model/Model.java:160]: Sorry this is because of the merging isn't it. My bad.

**Review by PotatoCombat — APPROVED** [review]
Will need to make a separate issue for the review comments.

File: src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.CommandWords.DELETE_MODEL;
+import static seedu.address.logic.commands.CommandWords.SESSION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ENDTIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MOD_CODE;
```
Comment (aakanksha-rai) [inline src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java]: Can you change PREFIX_MOD_CODE to PREFIX_MODULE as we had discussed last Monday?

File: src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java (hunk 4 above)
Comment (aakanksha-rai) [reply src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java:270]: Yeah same, I think these should not be deleted?

**Review by aakanksha-rai — COMMENTED** [review]
Mostly good, needs some small changes. Thanks!

File: src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java (hunk 5 above)
Comment (PotatoCombat) [reply src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java]: Let's do this in a separate PR!

**Pull request merged**